### PR TITLE
Add Integration guide for Okta

### PR DESCRIPTION
**Pull request by ismailmustafa**
### Description

Adds an Okta Integration Guide for our Next Generation Admin Console

### CODE REVIEW

File: docs/auth-sso/okta-integration-guide.mdx [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+
+<Arcade clip={Clip.OktaConfigureUserSettings} />
+
+### 🟠 Create a Beyond Identity Group
+- Create a group and add users to it
+
+TODO: Add arcade here
+
+### 🟠 Generate API Token
+- Security -> API -> Tokens -> Create token
```
Comment (shansmoore) [inline docs/auth-sso/okta-integration-guide.mdx:88]: Add "Go to" and a period to this sentence. Also change this to numbered steps

1. Go to Security -> API -> Tokens -> Create token.
2. Keep

File: docs/auth-sso/okta-integration-guide.mdx [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+
+## Set Up Beyond Identity
+### 🔵 Create Beyond Identity Account
+- Go to marketing website
+- Create BI account
+
+TODO: Add arcade here
+
+### 🔵 Integrate with Okta Directory
+- Go to directory integrations -> Add Directory -> Okta
```
Comment (shansmoore) [inline docs/auth-sso/okta-integration-guide.mdx:103]: Change these to numbered steps and add periods to the end of the sentences. 
Also, capitalize "Directory Integrations" and lowercase "Add directory"

File: docs/auth-sso/okta-integration-guide.mdx [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+### 🟠 Set Up OIDC Identity Provider
+- Set up OIDC identity provider.
+- Copy values from BI.
+- Save Identity provider
+- Set up routing rules for byndidRegistered attribute
+
+TODO: Add arcade here
+
+---
+
```
Comment (shansmoore) [inline docs/auth-sso/okta-integration-guide.mdx:138]: Should there be a section to test this with a small set of users?

**Review by shansmoore — COMMENTED** [review]
This looks great so far!

File: docs/auth-sso/okta-integration-guide.mdx [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+import Arcade, {Clip} from '@site/src/components/Arcade.tsx';
+
+# Okta Integration Guide
+
+This guide is designed to offer step-by-step instruction for integration Beyond Identity with your Okta environment, enabling a passwordless authentication solution. Specifically, you will learn how to:
+
+- Configure Beyond Identity as the primary passwordless authentication method for your Okta environment.
+- Establish Okta to recognize Beyond Identity as an Identity Provider, enhancing security and user experience.
```
[the remark this answers is not in the document]
Comment (ismailmustafa) [reply docs/auth-sso/okta-integration-guide.mdx]: 7e783009b95b2a7e8179183729a53c05bab45147

File: docs/auth-sso/okta-integration-guide.mdx [hunk 5]
```diff
@@ -0,0 +1,162 @@
...
+To enable Okta to automatically direct users registered with Beyond Identity towards a passwordless authentication flow, define a custom attribute named "byndidRegistered" on an Okta User's profile.
+
+<Arcade clip={Clip.OktaConfigureUserSettings} />
+
+### 🟠 Create a Beyond Identity Group
+- Create a group and add users to it
+
+TODO: Add arcade here
+
+### 🟠 Generate API Token
```
[the remark this answers is not in the document]
Comment (ismailmustafa) [reply docs/auth-sso/okta-integration-guide.mdx]: 7e783009b95b2a7e8179183729a53c05bab45147

File: docs/auth-sso/okta-integration-guide.mdx [hunk 6]
```diff
@@ -0,0 +1,162 @@
...
+- Security -> API -> Tokens -> Create token
+- Keep this somewhere safe for now
+
+TODO: Add arcade here
+
+---
+
+## Set Up Beyond Identity
+### 🔵 Create Beyond Identity Account
```
[the remark this answers is not in the document]
Comment (ismailmustafa) [reply docs/auth-sso/okta-integration-guide.mdx]: 7e783009b95b2a7e8179183729a53c05bab45147

File: docs/auth-sso/okta-integration-guide.mdx [hunk 7]
```diff
@@ -0,0 +1,162 @@
...
+## Prepare Okta for Integration
+
+### 🟠 Update User Profile Attributes
+
+To enable Okta to automatically direct users registered with Beyond Identity towards a passwordless authentication flow, define a custom attribute named "byndidRegistered" on an Okta User's profile.
+
+<Arcade clip={Clip.OktaConfigureUserSettings} />
+
+### 🟠 Create a Beyond Identity Group
+- Create a group and add users to it
```
[the remark this answers is not in the document]
Comment (ismailmustafa) [reply docs/auth-sso/okta-integration-guide.mdx]: Will be removing these bullet points once I add an Arcade.

File: docs/auth-sso/okta-integration-guide.mdx [hunk 8]
```diff
@@ -0,0 +1,162 @@
...
+
+TODO: Add arcade here
+
+### 🟠 Generate API Token
+- Security -> API -> Tokens -> Create token
+- Keep this somewhere safe for now
```
[the remark this answers is not in the document]
Comment (ismailmustafa) [reply docs/auth-sso/okta-integration-guide.mdx]: Will be removing these bullet points once I add an Arcade.

File: docs/auth-sso/okta-integration-guide.mdx [hunk 9]
```diff
@@ -0,0 +1,162 @@
...
+
+TODO: Add arcade here
+
+---
+
+## Set Up Beyond Identity
+### 🔵 Create Beyond Identity Account
+- Go to marketing website
```
[the remark this answers is not in the document]
Comment (ismailmustafa) [reply docs/auth-sso/okta-integration-guide.mdx:98]: Will be removing these bullet points once I add an Arcade.

File: docs/auth-sso/okta-integration-guide.mdx (hunk 1 above)
Comment (ismailmustafa) [reply docs/auth-sso/okta-integration-guide.mdx]: Will be removing these bullet points once I add an Arcade.

File: docs/auth-sso/okta-integration-guide.mdx (hunk 2 above)
Comment (ismailmustafa) [reply docs/auth-sso/okta-integration-guide.mdx:104]: Will be removing these bullet points once I add an Arcade.

File: docs/auth-sso/okta-integration-guide.mdx (hunk 3 above)
Comment (ismailmustafa) [reply docs/auth-sso/okta-integration-guide.mdx:139]: Yes good idea. I've added a note to include that. Thank you.

**Pull request merged**